src/main/java/software/amazon/qldb/QldbDriverBuilder.java

```diff
@@ -15,6 +15,7 @@
 
 import com.amazon.ion.IonSystem;
 import java.util.concurrent.ExecutorService;
+import software.amazon.awssdk.http.SdkHttpClient;
 import software.amazon.awssdk.services.qldbsession.QldbSessionClientBuilder;
 
 /**
```

Comment:
Do we need to update the snippet? We can build with a customized http client with max connections configured, which is helpful to differentiate with pool limit.

Reply:
Updated